devtools/__init__.py

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/env python3
+
+import sys, os, subprocess, shlex, typing
+from typing import Mapping, Dict, Optional
+
+GITHUB_ACTIONS = bool(os.environ.get('GITHUB_ACTIONS'))
+OFFICIAL_CONTAINER_REPO = 'ghcr.io/evalf/nutils'
```

Comment:
perhaps it's cleaner to move this to `container/__init__.py`?

Reply:
Done.